save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -141,4 +141,12 @@ class VulnerabilityService(
             )
         }
     }
+
+    /**
+     * @param name name of project
+     * @param vulnerabilityName name of vulnerability
+     * @return Unit
```

Comment:
Is it required?

Reply:
ahh, yes. because it's an expression body